doc/examples/parallel_planning/config/parallel_planning_moveit_cpp.yaml

```diff
@@ -0,0 +1,38 @@
+planning_scene_monitor_options:
```

Comment:
I would consider adding brief comments on top of all these sections, as these are a tutorial and you want to explain a bit more than normally.